ftest-maven/quarkus-example/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,22 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+#
+# Before building the docker image run:
+#
+# mvn package
+#
+# Then, build the image with:
+#
```

Comment:
why is this dockerfile required?

Reply:
Comes from Quarkus, I left as it is always generated by Quarkus archetype.

Reply:
ohk